kola/tests/docker/docker.go

```diff
@@ -610,3 +610,21 @@ func testDockerInfo(expectedFs string, c cluster.TestCluster) {
 		c.Errorf("runc was not in runtimes: %+v", info.Runtimes)
 	}
 }
+
+// hasSecurityOptions strictly checks that at least one of
+// the docker security option is enabled (seccomp, selinux)
+func hasSecurityOptions(opts []string) bool {
+	var selinux, seccomp bool
+	for _, opt := range opts {
+		switch opt {
+		case "selinux":
+			selinux = true
+		case "seccomp":
+			seccomp = true
+		default:
+			return false
+		}
+	}
+
+	return selinux || seccomp
```

Comment:
Maybe you can simplify to:
```suggestion
	for _, opt := range opts {
		switch opt {
		case "selinux", "seccomp":
			return true
		}
	}

	return false
```

Reply:
Good idea but I'm not sure we won't strictly check for `selinux`, `seccomp` or both. Let's suppose `opts = ["apparmor", "selinux"]` we will return `true` even if we have `apparmor` in the options. :thinking:

Reply:
so maybe:
```go
switch opt {
case "selinux", "seccomp":
default:
  return false
}

return true
```